examples/mach_eval_examples/SynR_eval/SynR_em_post_analyzer.py

```diff
@@ -103,8 +115,16 @@ def get_next_state(results, in_state):
 
         state_out.conditions.em = post_processing
 
+        print("\n************************ ELECTROMAGNETIC LOSSES ************************")
+        print("Stator Ohmic = ", stator_calc_ohmic_loss, " W",)
+        print("Windage = ", windage_loss, " W")
+        print("Stator Iron = ", stator_hysteresis_loss, " W",)
+        print("Rotor Iron = ", rotor_hysteresis_loss, " W")
+        print("*************************************************************************\n")
+
         print("\n************************ ELECTROMAGNETIC RESULTS ************************")
         #print("Torque = ", torque_avg, " Nm")
+        print("Speed-Power = ", omega_m*np.sqrt(P_out), " RPM/kW") 
```

Comment:
Shouldn't the unit here be `RPM * sqrt(W)`? Or `RPM * sqrt(kW)` if you change the unit for `P_out` to be `kW`.

Reply:
You are right, thanks for catching that.